src/Sidekick.Domain/Game/StatTranslations/AlternateStatProvider.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+
+namespace Sidekick.Domain.Game.StatTranslations
+{
+    public class AlternateStatProvider : IAlternateStatProvider
+    {
+        public AlternateStatProvider()
+        {
+            Translations = new List<StatTranslation>();
+
+            var assembly = this.GetType().GetTypeInfo().Assembly;
+            using Stream resource = assembly.GetManifestResourceStream("Sidekick.Domain.Game.StatTranslations.stat_translations.min.json");
+
+            using var stream = new StreamReader(resource);
+            var data = stream.ReadToEnd();
+            Translations = JsonSerializer.Deserialize<List<StatTranslation>>(data);
+
+            foreach (var translation in Translations)
+                CleanTranslation(translation);
+
+            Translations = Translations.Where(x => x.Stats.Any(y => y.Conditions != null && y.Conditions.Count >= 1)).ToList();
```

Comment:
This should be in an Initialize method. Initialize methods will allow the usage of Async methods instead of using the Synchonous ones that you must use in a constructor.